src/reducers/profile.js

```diff
@@ -292,7 +292,6 @@ function onAddWebLinkDone(state, { payload, error }) {
 
   if (error) {
     logger.error('Failed to add web link', payload);
-    fireErrorMessage('ERROR: Failed to add web link!');
```

Comment:
Why to remove UI error message in case of error? Without it user will have no idea, whether he should wait more, or operation failed, etc.

Reply:
https://github.com/topcoder-platform/community-app/blob/f94c9106fa3719e003256726ad1b8a9364fe9124/src/shared/reducers/page/settings.js#L87

because i added new error message keep behavior like existing code, https://monosnap.com/file/uNhRA5ZhBzOWXKwVCYXZDm7ZAiTVHp
you can confirm this add new weblink at https://www.topcoder.com/settings/profile/ 

fireErrorMessage will throw a global error message, that not looks good for user

Reply:
We can bend existing look and behavior to follow newer TC styles, and codebase. The problem is that doing so, you ask every user of _profile_ actions / reducers to care himself about error reporting in the pages where these actions are used. And the original approach automatically ensures error reporting.